### PR TITLE
Added support for async await

### PR DESCRIPTION
**Pull request by dw1284**

### CODE REVIEW

**Review by uzairfarooq — CHANGES_REQUESTED** [review]
Can you please send the pull request to dev branch?

File: src/arrive.js [hunk 1]
```diff
@@ -355,13 +354,22 @@ var Arrive = (function(window, $, undefined) {
...
         }
 
         setTimeout(utils.callCallbacks, 1, existing);
       }
 
-      mutationBindEvent.call(this, selector, options, callback);
+      if (callback) {
+        mutationBindEvent.call(this, selector, options, callback);
+      } else {
+        var a = this;
```
[the remark this answers is not in the document]
Comment (dw1284) [reply src/arrive.js:370]: Yes exactly. The promise was rescoping this so I had to send it in another way.

**Comment by dw1284** [comment]
> Can you please send the pull request to dev branch?

I updated the target branch of the PR to dev

**Review by uzairfarooq — APPROVED** [review]

**Comment by uzairfarooq** [comment]
Go ahead and merge (Squash and merge). I will release it afterwards.

**Comment by uzairfarooq** [comment]
Never mind, I merged it. Will release it in a day or two.

**Comment by dw1284** [comment]
Awesome, thanks!

**Pull request merged**